bin/makefile-openbsd.x86_64-x

```diff
@@ -1,8 +1,6 @@
 # Options for OpenBSD, Intel x86_64 and X-Window
 
-#CC = gcc -m64 $(GCC_CFLAGS) -fno-omit-frame-pointer -Wall -Wextra -fno-aggressive-loop-optimizations
-CLANG_CFLAGS += -I/usr/X11R6/include
-CC = clang -m64 $(CLANG_CFLAGS)
+CC = gcc -m64 $(GCC_CFLAGS)
```

Comment:
Isn't Clang the default compiler on OpenBSD these days?  I'd pick whichever one is installed by default while choosing clang over gcc if they're both there.

Reply:
Yes, that's probably the better way to go.